mailpoet/assets/js/src/automation/automation.tsx

```diff
@@ -27,7 +56,14 @@ function Content(): JSX.Element {
 
   const workflows = data?.data ?? [];
   return workflows.length > 0 ? (
-    <AutomationListing workflows={workflows} loading={loading} />
+    <>
+      {notices}
```

Comment:
One thing to note is that this file  now has two different ways of generating notices: one based on URL params and this being the other. Once we've settled on an approach I was hoping to consolidate these to use the same approach.

Reply:
Would be great to consolidate them to `@wordpress/notices`. Some may need params from URL because they're being rendered after a redirect though.